sonar-plugin-api/src/test/java/org/sonar/api/server/ws/RequestTest.java

```diff
@@ -214,6 +222,21 @@ public void param_as_date() {
     assertThat(underTest.setParam("a_date", "2014-05-27").paramAsDate("a_date")).isEqualTo(DateUtils.parseDate("2014-05-27"));
   }
 
+  @DataProvider
+  public static Object[][] date_times() {
```

Comment:
I understand. It's because I initially tried to work on Request before putting methods in DateUtils. What bothers me is that you need to specify a Runner and you can have only one by test class